### DISCUSSION TITLE
Cannot find module 'radix-vue/date'

### DISCUSSION THREAD

**Discussion by delvin02** [unovue/reka-ui#942] [General]
I'm trying to integrate [Date Range Picker with Independent Months](https://www.shadcn-vue.com/docs/components/date-picker.html#date-range-picker-with-independent-months) but I'm encountering an unexpected library import error with the following code:

import { type Grid, createMonth, toDate } from 'radix-vue/date';

could it be an implementation error on the documentation site?

**Comment by zernonia** [comment]
@delvin02  can you share what issue you are facing?

**Comment by delvin02** [comment]
![image](https://github.com/radix-vue/radix-vue/assets/64721581/87b3d5f9-a0ca-4b66-82af-673909b6ea56)

I got this issue when importing from 'radix-vue/date' as shown in the picture. Still couldn't figure out what the error is about

any help is appreciated!

**Reply by zernonia** [reply]
Have you tried updating the `moduleResolution` to the suggested option?

**Reply by delvin02** [reply]
I tried that, it does not help.

I resolved the issue by adding the path
![image](https://github.com/radix-vue/radix-vue/assets/64721581/4d541f37-f760-42cb-bc4b-4b7740205da2)

for whatever reason, when I start building it, it shows me errors from my shadcn UI component library. Do you have any idea on this? 

![image](https://github.com/radix-vue/radix-vue/assets/64721581/667b2def-634c-4677-a884-c6eea65790af)

**Reply by zernonia** [reply]
@delvin02  I saw your config has `moduleResoltion: "node"`, and indeed if I set that there's the warning shows.
However changing to `moduleResolution: "Bundler"` solves the issue. Can you give it a try?

**Reply by delvin02** [reply]
oh yes, it works beautifully now!

thanks for the work in shadcn vue, it's awesome!

**Reply by zernonia** [reply]
I'm glad to hear that! 💚
